src/main/java/uk/gov/hmcts/reform/sscs/json/RoboticsJsonMapper.java

```diff
@@ -111,6 +111,10 @@ private static JSONObject buildHearingOptions(HearingOptions hearingOptions) {
 
             hearingArrangements.put("hearingLoop", convertBooleanToYesNo(hearingOptions.wantsHearingLoop()));
             hearingArrangements.put("accessibleHearingRoom", convertBooleanToYesNo(hearingOptions.wantsAccessibleHearingRoom()));
+        } else if (hearingOptions.getOther() != null || hearingOptions.getExcludeDates() != null) {
+
+            hearingArrangements.put("hearingLoop", convertBooleanToYesNo(false));
```

Comment:
This logic does not look right to me. This is going to overwrite whatever the user enters into the hearingLoop and accessibleHearingRoom fields and set to false. I think we only want to overwrite it if those values do not exist. We can pair on Monday if that is not clear

Reply:
hearingLoop and accessibleHearingRoom properties are in arrangements only so if arrangements is not null then it check values. if arrangements is null then it will be default to false

Reply:
ok understood - looks good